### PR TITLE
AC Test Fix - LRAC-9441 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LRAC-9441 (fix ticket)
https://issues.liferay.com/browse/LRAC-9438 (refactor)
https://issues.liferay.com/browse/LRAC-9439 (refactor)
[afonsohsantos#23](https://github.com/AfonsoHSantos/liferay-portal/pull/23)
https://test-1-15.liferay.com/job/publish-testray-report/1831/ (report)

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase [hunk 1]
```diff
@@ -238,6 +245,11 @@ definition {
 	@description = "Automation ID: LRAC-8892 | Test Summary: Suppressed user appears in Suppressed User List"
 	@priority = "5"
 	test SuppressedUserAppearsInSuppressedUserList {
+		property portal.acceptance = "quarantine";
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase:248]: Hi @AfonsoHSantos

Correct:
`property portal.upstream = "quarantine";`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase [hunk 2]
```diff
@@ -183,6 +183,11 @@ definition {
 	@description = "Automation ID: LRAC-8893 | Test Summary: Able to unsuppress user"
 	@priority = "3"
 	test CanUnsuppressUser {
+		property portal.acceptance = "quarantine";
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase:186]: Hi @AfonsoHSantos

Correct:
`property portal.upstream = "quarantine";`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase (hunk 1 above)
Comment (AfonsoHSantos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase]: mb, fixed

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase (hunk 2 above)
Comment (AfonsoHSantos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase]: mb, fixed

**Pull request closed without merging**